src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2023,6 +2036,72 @@ export class StudyViewPageStore {
         }
     }, []);
 
+    readonly oncokbCancerGenes = remoteData<CancerGene[]>({
+        await: () => [],
+        invoke: async () => {
+            return oncoKBClient.utilsCancerGeneListGetUsingGET({});
+        },
+        onError: (error => {
+            console.log(error);
```

Comment:
console.log.    If we specify onError, we disable the global handling of this error (error screen) so we should make sure that we handle it properly in code.